docs/releases/2.0.txt

```diff
@@ -228,6 +228,8 @@ Database backend API
   ``datetime_cast_time_sql()``, ``datetime_trunc_sql()``, and
   ``datetime_extract_sql()`` methods now return only the SQL to perform the
   operation instead of SQL and a list of parameters.
+* DatabaseWrapper now pass through args and kwargs to underlying database
```

Comment:
Needs a better explanation and should link to the section that will show the example.